_pytest/terminal.py

```diff
@@ -42,6 +42,10 @@ def pytest_addoption(parser):
                      action="store", dest="tbstyle", default='auto',
                      choices=['auto', 'long', 'short', 'no', 'line', 'native'],
                      help="traceback print mode (auto/long/short/line/native/no).")
+    group._addoption('--show-capture',
+                     action="store", dest="showcapture",
+                     choices=['no', 'stdout', 'stderr'],
+                     help="Print only stdout/stderr on not print both.")
```

Comment:
This would read better IMHO: `controls how captured stdout/stderr is shown on failed tests. default is "both".`